libcontainer/intelrdt/intelrdt.go

```diff
@@ -397,7 +397,7 @@ func writeFile(dir, file, data string) error {
 	if dir == "" {
 		return fmt.Errorf("no such directory for %s", file)
 	}
-	if err := ioutil.WriteFile(filepath.Join(dir, file), []byte(data+"\n"), 0700); err != nil {
+	if err := ioutil.WriteFile(filepath.Join(dir, file), []byte(data+"\n"), 0o644); err != nil {
```

Comment:
Strictly speaking, file mode is not used here as the file is *not* being created, it's already there.

But I'd prefer `0o600` here.

Reply:
`0o` isn't commonly used in Go (or C) for octal values, just use `0644` -- same goes for the other changes.

Reply:
@kolyshkin 

Thank you for reviewing the commits.

>Strictly speaking, file mode is not used here as the file is not being created, it's already there.

Yes.

>But I'd prefer 0o600 here.

As you know, This file is automatically created as `0o644` and it is not changed even if we specify `0o600`.
So I think we don't need to change the default `0o644` .

And is it ok to change the format from `0o` to `0`? If yes, I will modify the format again.
Either is fine for me. But I just want to unify the format as we discussed in below
https://github.com/opencontainers/runc/pull/2625#issuecomment-702627945

Reply:
> 0o isn't commonly used in Go (or C) for octal values, just use 0644 -- same goes for the other changes.

It's a new thing as of Go 1.13, and I think it makes sense to use it as it is more explicit.